lms/djangoapps/grades/tests/test_new.py

```diff
@@ -239,60 +242,162 @@ def test_save_and_load(self):
 
 
 @ddt.ddt
-class TestMultipleProblemTypesSubsectionScores(ModuleStoreTestCase, ProblemSubmissionTestMixin):
+class TestMultipleProblemTypesSubsectionScores(SharedModuleStoreTestCase):
     """
     Test grading of different problem types.
     """
 
-    default_problem_metadata = {
-        u'graded': True,
-        u'weight': 2.5,
-        u'max_score': 7.0,
-        u'due': datetime.datetime(2099, 3, 15, 12, 30, 0, tzinfo=pytz.utc),
-    }
+    SCORED_BLOCK_COUNT = 7
+    ACTUAL_TOTAL_POSSIBLE = 16.0
 
-    COURSE_NAME = u'Problem Type Test Course'
-    COURSE_NUM = u'probtype'
+    @classmethod
+    def setUpClass(cls):
+        super(TestMultipleProblemTypesSubsectionScores, cls).setUpClass()
+        cls.load_scoreable_course()
+        chapter1 = cls.course.get_children()[0]
+        cls.seq1 = chapter1.get_children()[0]
 
     def setUp(self):
         super(TestMultipleProblemTypesSubsectionScores, self).setUp()
         password = u'test'
         self.student = UserFactory.create(is_staff=False, username=u'test_student', password=password)
         self.client.login(username=self.student.username, password=password)
         self.request = get_request_for_user(self.student)
-        self.course = CourseFactory.create(
-            display_name=self.COURSE_NAME,
-            number=self.COURSE_NUM
+        self.course_structure = get_course_blocks(self.student, self.course.location)
+
+    @classmethod
+    def load_scoreable_course(cls):
+        """
+        This test course lives at common/test/data/scoreable.
+
+        It contains six scoreable blocks in one sequence:
```

Comment:
Nit: Should this description about the course content locally live with its directory (within common/test/data/scoreable)?  Maybe in a README file?